cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -1,4 +1,4 @@
-describe('Creator Setup Page', () => {
+describe('Creator Settings Page', () => {
```

Comment:
Just changed the naming to align with the route and file name

Reply:
Great catch!